prov/verbs/src/verbs_eq.c

```diff
@@ -164,6 +165,9 @@ fi_ibv_eq_cm_process_event(struct fi_ibv_eq *eq, struct rdma_cm_event *cma_event
 	case RDMA_CM_EVENT_ESTABLISHED:
 		*event = FI_CONNECTED;
 		entry->info = NULL;
+		ret = fi_ibv_modify_qp(cma_event->id->qp);
```

Comment:
If I correctly know, EQ isn't required in case of RDM EP type.
Maybe move it to fi_ibv_rdm_pack_cm_params for RDM type,  fi_ibv_msg_ep_connect and fi_ibv_msg_ep_accept for MSG type.
What do you think regarding proposed changes?